db/seeds.rb

```diff
@@ -0,0 +1,29 @@
+require "bundler/setup" # require all the gems we'll be using for this app from the Gemfile. Obviates the need for `bundle exec`
+require "pg" # postgres db library
+require "active_record" # the ORM
+require "pry" # for debugging
+
+require_relative "connection"
+require_relative "../models/apartments"
+require_relative "../models/tenants"
+
+Tenant.destroy_all
+Apartment.destroy_all
+
+
+
+zanehs_house = Apartment.create(address: "29 Fairhill Lane", monthly_rent: 200, sqft: 3000, num_beds: 3, num_baths: 4)
+grandmas_house = Apartment.create(address: "6508 Springbrook Lane", monthly_rent: 400, sqft: 3200, num_beds: 4, num_baths: 6)
+random_house = Apartment.create(address: "670 Fairy Lane", monthly_rent: 900, sqft: 2000, num_beds: 2, num_baths: 1)
+
+Tenant.create(name: "Zaneh", age: 23, gender: "female", apartment_id: zanehs_house[:id])
```

Comment:
This works, but since the house variables hold AR Instances / objects, you can use the *dot* syntax to call the id method instead of brackets:

```
zanehs_house.id
```